server/internal/store/usage.go

```diff
@@ -0,0 +1,23 @@
+package store
+
+import "gorm.io/gorm"
+
+// Usage represents a API usage.
+type Usage struct {
+	gorm.Model
+
+	User         string
```

Comment:
Just wanted to make sure we're on the same page. We will work on indexing later?